client/src/components/layout/Header.js

```diff
@@ -13,7 +13,7 @@ import { Fragment } from "react";
  * @returns Header component to be rendered in Layout.js
  */
 const Header = ({title, urls}) => {
```

Comment:
Static Analyzer warnings, unused imports.

Reply:
I wasn't able to target the affected lines in my comment above, but the unused imports are in this file.